packages/eas-cli/src/submit/ArchiveSource.ts

```diff
@@ -304,42 +305,36 @@ async function handleBuildListSourceAsync(
 function formatBuildChoice(build: BuildFragment, expiryDate: Date): prompts.Choice {
   const {
     id,
-    platform,
     updatedAt,
-    appVersion,
-    sdkVersion,
     runtimeVersion,
     buildProfile,
-    appBuildVersion,
     releaseChannel,
-    initiatingActor,
+    gitCommitHash,
+    gitCommitMessage,
   } = build;
 
   const formatValue = (field?: string | null): string =>
     field ? chalk.bold(field) : chalk.dim('Unknown');
 
   const buildDate = new Date(updatedAt);
-  const maybeRuntimeVersion = runtimeVersion ? `Runtime: ${formatValue(runtimeVersion)}` : null;
-  const maybeSdkVersion = sdkVersion ? `SDK: ${formatValue(sdkVersion)}` : null;
-  const appBuildVersionString = `${
-    platform === AppPlatform.Android ? 'Version code' : 'Build number'
-  }: ${formatValue(appBuildVersion)}`;
-
-  const title = [
-    `ID: ${chalk.dim(id)}, Finished at: ${chalk.bold(buildDate.toLocaleString())}`,
-    [
-      `\tApp version: ${formatValue(appVersion)}, ${appBuildVersionString}`,
-      maybeRuntimeVersion,
-      maybeSdkVersion,
-    ]
-      .filter(it => it != null)
-      .join(', '),
-    `\tProfile: ${formatValue(buildProfile)}, Release channel: ${formatValue(releaseChannel)}`,
-    `\tAuthored by: ${formatValue(initiatingActor?.displayName)}`,
+
+  const formattedCommitData =
+    gitCommitHash && gitCommitMessage
+      ? `${chalk.dim(gitCommitHash.slice(0, 7))} "${chalk.bold(gitCommitMessage)}"`
+      : 'Unknown';
+
+  const title = `ID: ${chalk.dim(id)} (${chalk.dim(`${fromNow(buildDate)} ago`)})`;
+
+  const description = [
+    `\tProfile: ${formatValue(buildProfile)}`,
+    `\tChannel: ${formatValue(releaseChannel)}`,
```

Comment:
Is it "release channel" (legacy) or "channel" (eas update)? If the former, then maybe we should name this "Release channel"? Also, maybe it's worth printing the "channel" as well?

Reply:
Thanks, I meant to use EAS Update `build.channel` as data for `Channel`. We want to switch to only using the `Channel` vs `Release Channel`.